security-admin/scripts/setup.sh

```diff
@@ -747,9 +748,9 @@ update_properties() {
 	if [ "${DB_FLAVOR}" == "MSSQL" ]
 	then
 		propertyName=ranger.jpa.jdbc.url
-		if [ "${DOCKER_ENV}" == "true" ]
+		if [ "${CONNECTION_STRING_ADDITIONAL_PARAMS}" != "" ]
 		then
-  			newPropertyValue="jdbc:sqlserver://${DB_HOST};databaseName=${db_name};trustServerCertificate=true;"
+  			newPropertyValue=${CONNECTION_STRING_ADDITIONAL_PARAMS}
```

Comment:
`newPropertyValue="jdbc:sqlserver://${DB_HOST};databaseName=${db_name};${CONNECTION_STRING_ADDITIONAL_PARAMS}`"

Reply:
Also, add the following to install.properties used in docker setup:
`CONNECTION_STRING_ADDITIONAL_PARAMS="trustServerCertificate=true;"`